notebooks/timelapse.ipynb

```diff
@@ -263,7 +264,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "Image(filename='{}/{}'.format(timelapse_dir, 'timelapse_simple.gif'), width = 500)"
+    "Image(filename=os.path.join(TIMELAPSE_DIR, 'timelapse_simple.gif'), width = 500)"
```

Comment:
In cases like this perhaps `file_path = os.path.join(TIMELAPSE_DIR, 'timelapse_simple.gif')` could be written in a separate line for more clarity.

Also, no spaces at `width = 500`.